tests/test_db.py

```diff
@@ -171,21 +170,16 @@ def test_scid_upgrade(node_factory, bitcoind):
 def test_last_tx_inflight_psbt_upgrade(node_factory, bitcoind):
     bitcoind.generate_block(12)
 
-    prior_txs = ['02000000019CCCA2E59D863B00B5BD835BF7BA93CC257932D2C7CDBE51EFE2EE4A9D29DFCB01000000009DB0E280024A01000000000000220020BE7935A77CA9AB70A4B8B1906825637767FED3C00824AA90C988983587D68488F0820100000000002200209F4684DDB28ACDC73959BC194D1A25DF906F61ED030F52D163E6F1E247D32CBB9A3ED620', '020000000122F9EBE38F54208545B681AD7F73A7AE3504A09C8201F502673D34E28424687C01000000009DB0E280024A01000000000000220020BE7935A77CA9AB70A4B8B1906825637767FED3C00824AA90C988983587D68488F0820100000000002200209F4684DDB28ACDC73959BC194D1A25DF906F61ED030F52D163E6F1E247D32CBB9A3ED620']
+    # FIXME: Re-add dynamic checks once PSBTv2 support is in both Core/Elements, or get python support
+    # These PSBTs were manually checked for 0.001 BTC multisig witness utxos in a single input
```

Comment:
There is a middle road I guess - you can keep the dynamic checks with minimal test changes  if you bring in wally but just use it to downgrade to v0 before decoding with core.